model/models.py

```diff
@@ -128,3 +128,35 @@ def forward(self, sub, rel):
 
 		score = torch.sigmoid(x)
 		return score
+
+class CompGCN_ConvKB(CompGCNBase):
+	'''Implemtents the ConvKB Scoring Function'''
+	def __init__(self, edge_index, edge_type, params=None):
+		super(self.__class__, self).__init__(edge_index, edge_type, params.num_rel, params)
+		self.drop = torch.nn.Dropout(self.p.hid_drop)
+		self.relu = torch.nn.ReLU()
+		self.conv2d1 = torch.nn.Conv2d(1, out_channels=self.p.num_filt, kernel_size=(1, 3), stride=1)
+		self.fake_fc_with_conv = torch.nn.Conv2d(self.p.num_filt,out_channels= 1 , kernel_size=(self.p.embed_dim, 1), stride=self.p.embed_dim,bias=False)
+
+		#Prints the number of parameter in the network to the console
+		total_params = sum(
+			param.numel() for param in self.parameters()
+		)
+		print("Total Parameter: " + str(total_params))
+
+	def forward(self, sub,rel):
```

Comment:
could you please add some docstring explaining this function?

Reply:
added documentation

Reply:
ok